src/ComposerRequireChecker/DefinedSymbolsLocator/LocateDefinedSymbolsFromASTRoots.php

```diff
@@ -3,14 +3,15 @@
 namespace ComposerRequireChecker\DefinedSymbolsLocator;
 
 use ComposerRequireChecker\NodeVisitor\DefinedSymbolCollector;
+use PhpParser\Node;
 use PhpParser\NodeTraverser;
 use PhpParser\NodeVisitor\NameResolver;
 use Traversable;
 
 final class LocateDefinedSymbolsFromASTRoots
 {
     /**
-     * @param Traversable $ASTs a series of AST roots
+     * @param Traversable<int, array<Node|null>|mixed> $ASTs a series of AST roots
```

Comment:
`mixed` also here defeats the other types

Reply:
well see my comment 🗡️ 
You are just to fast with the review 😆